### ISSUE TITLE
Add an option to change the file extension of generated e-books

### ISSUE THREAD

**Issue by leotaku** [leotaku/kojirou#47]
Here is my experience with this mobi format. I used to own a Kindle but now I just purchased a Boox device so I need mobi files for the reading app. 
I have looked around GitHub and tbh Kojirou is by far the best Mangadex downloader I have ever tried. But the awz3 format doesn't work for me so I decided to change the file extension to .mobi and it works like a charm. 
I think you can add one option for the output to be saved in .mobi instead and this can be used for every single e-reader on earth.

_Originally posted by @baobach in https://github.com/leotaku/kojirou/issues/46#issuecomment-2542720831_

**Comment by sajilal711** [comment]
Hey , great project 
I too want output in mobi files, and am sseing some mobi.go files.  how to trigger this via the cli? If not yet implemented just want to mention looking forward to it.
Mainly want mobi for the covers and lock screen to work properly on kindle.
Regards

**Comment by leotaku** [comment]
@sajilal711 Hi! Yeah, unfortunately I kind of forgot about implementing this. Do you know if just renaming the files from azw3 to mobi theoretically fixes the issues for you? Because if so (as mentioned in the original issue) I can simply add a switch to configure the file extension.

**Comment by sajilal711** [comment]
Hi @leotaku I checked ny just changing the file extension, it shows as unreadable and only option is to delete.
Btw i am sideloading via usb-c to documents folder, are you loading using send to kindle? like are you able to see the covers ? well it is a nitpick at this point, anyways i have over ridden the default lockscreen wallpapers with custom ones lol.

**Comment by leotaku** [comment]
Yeah, that's the thing.  Naming the file AZW3 seems to result in it being compatible with the largest number of devices, but there are some quirks. AZW3 should just be a superset of MOBI, but, as you have noticed, lock screen covers do not seem to work.

I mostly transfer my files using Calibre, but I also don't have lock screen covers working on my device currently. I seem to recall that converting to MOBI using Calibre might have resulted in working lock-screen covers at some point, but I would have to test it again.
